diff --git a/include/FileClasses/TextManager.h b/include/FileClasses/TextManager.h
index cabf665..873fabe 100644
--- a/include/FileClasses/TextManager.h
+++ b/include/FileClasses/TextManager.h
@@ -18,16 +18,17 @@
 #ifndef TEXTMANAGER_H
 #define TEXTMANAGER_H
 
-#include "IndexedTextFile.h"
-#include "MentatTextFile.h"
-
 #include <string>
 #include <vector>
 #include <map>
 #include <memory>
+#include <array>
 
 #include <algorithm>
 
+#include "IndexedTextFile.h"
+#include "MentatTextFile.h"
+
 #define MISSION_DESCRIPTION     0
 #define MISSION_WIN             1
 #define MISSION_LOSE            2
